NETPROGRAM/ArrayExercise/Exercise04.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NETPROGRAM.ArrayExercise
+{
+    public class Exercise04
+    {
+
+        public bool IsPrime(int n)
+        {
+            if(n < 2)
+            {
+                return false;
+            }
+
+            for(var i = 2; i <= n / i; i++)
+            {
+                if(n % i == 0)
+                {
+                    return false;
+                }
+            }
+
+            return true;
+        }
+
+        public int SumIPrime(int[] array)
+        {
+
+            return array.Sum(x => IsPrime(x) ? x : 0);
```

Comment:
Dùng thế này cũng được nhưng không khuyến khích vì sai mục đích, ngữ nghĩa
```suggestion
            return array.Where(IsPrime).Sum();
```

Reply:
done